### DISCUSSION TITLE
Relay server, IP instead of FQDN, mx lookup errors on RFC1918 IP space

### DISCUSSION THREAD

**Discussion by 11notes** [stalwartlabs/smtp-server#43] [Q&A]
I checked the documentation, but couldn’t find how to disable MX record checks for a relay. Stalwart SMTP refuses to deliver the mail to the relay server because of missing MX record. Any idea? I can set the MX record, that's not the issue, but I would rather have the correct settings so it will connect to this relay, regardless of MX record since this is an internal RFC1918 connection via VPN and no MX lookup is needed.

**Comment by mdecimus** [comment]
MX records are not checked for relay hosts. How are you defining your relay host in the config?

**Reply by 11notes** [reply]
IP, RFC1918 IP.


```
[remote."local"]
address = "10.18.182.51"
port = 25
protocol = "smtp"
concurrency = 32
timeout = "1m"

[remote."local".tls]
implicit = false
allow-invalid-certs = true
```

**Reply by mdecimus** [reply]
Try adding the IP address to your `/etc/hosts` file and use a hostname instead. If that works please open an issue to support IP addresses in relay hosts.

**Reply by floralpetals** [reply]
I have the same issue, adding it to `/etc/hosts` didn't fix it.
